### ISSUE TITLE
Re-training a model

### ISSUE THREAD

**Issue by ijauregiCMCRC** [zhiguowang/BiMPM#3]
Is it possible to re-train a model? If so how?

**Comment by zhiguowang** [comment]
Yes, it supports re-train the model. If you want to re-train a model, just re-launch the same command line (the code will check if there is already a model in you path. if so, it will load the model and train based on the model).
For more details, can you have a look at lines 137-162 and lines 243-246 in SentenceMatchTrainer.py
